### PR TITLE
documentation for access control of shells

### PR DESCRIPTION
**Pull request by tunacicek**
## Description
Add documentation for access control of shells

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -420,93 +420,186 @@ Depending on being a Data Provider or a Data Consumer there are different tokens
...
+#### Postman configuration
+![](img/image005.png)
+
+*Support contact*	tractusx-dev@eclipse.org
+
+### Access control to Digital Twins Based on the BPN (Business Partner Number)/ TenantId
+The visibility of `specificAssetIds` in the Digital Twin Registry based on the Business Partner Number (BPN) can be controlled with the attribute `externalSubjectId`. Hence, the `externalSubjectId` is identified with the BPN.
+
+* The BPN as attribute to a *specificAssetId* can be optionally provided in `specificAssetIds`. This can be done with `externalSubjectId`.
+* Only those users, where `externalSubjectId` matches the Eclipse Dataspace Components-Header (*i.e.* BPN) are able to discover via and read exactly this content of `specificAssetIds`.
```
Comment (bs-sili) [inline docs/README.md:435]: Wouldn't make it more sense to to write this sentence "...  are able to discover and read ..."? So without "via"?

File: docs/README.md [hunk 2]
```diff
@@ -420,93 +420,186 @@ Depending on being a Data Provider or a Data Consumer there are different tokens
...
+* The behavior of the Digital Twin Registry is as follows. The read-access from `specificAssetIds` is inherited by the other fields of a *Digital Twin*.
+   * An owner and a user with respect to the BPN, who has read-access to one of the `specificAssetIds` (`externalSubjectId` has `"value":"<Business Partner Number>"`) has read-access to the other fields of a *Digital Twin* as well, *i.e.* (see the full list of fields [POST Asset Administration Shell Descriptor v3.0 SSP-001](https://app.swaggerhub.com/apis/Plattform_i40/AssetAdministrationShellRegistryServiceSpecification/V3.0_SSP-001#/Asset%20Administration%20Shell%20Registry%20API/PostAssetAdministrationShellDescriptor))
+      * `description`
+      * `displayName`
+      * `extensions`
```
Comment (bs-sili) [inline docs/README.md:443]: Shouldn't we delete the "extensions" as the DTR do not provide this field?

File: docs/README.md [hunk 3]
```diff
@@ -420,93 +420,186 @@ Depending on being a Data Provider or a Data Consumer there are different tokens
...
-- You can provide the tenantId/BPN as attribute to a specificAssetId. Only users having the same tenantId/BPN in the Header (property: Edc-Bpn) are able to see the specificAssetId.
-- The communication between consumer and provider is via EDC. Before the provider EDC sends the request to the DTR, the property Edc-Bpn with the BPN of the consumer will be set by the provider EDC.
-- The specificAssetIds of Digital Twins you created will always be shown to you.
+   * A user, who has <u>only</u> general read-access to `specificAssetIds` (`externalSubjectId` has `"value":"PUBLIC_READABLE"`) has <u>only</u> read-access to the following fields of a *Digital Twin*
+      * `id`
+      * `submodelDescriptors`
 
-Detailed example:
+**Hint** <br>
+The defined character for `"PUBLIC_READABLE"` and the list of allowed types to mark as public is configurable via helm values.yml:
```
Comment (thomas-henn) [inline docs/README.md:456]: The defined string...

**Review by thomas-henn — CHANGES_REQUESTED** [review]

**Comment by thomas-henn** [comment]
Behavior of Digital Twin Registry with respect to read access to digital twins is described here as aligned with the users of this application.

**Comment by thomas-henn** [comment]
Behavior of Digital Twin Registry with respect to read access to digital twins is described here as aligned with the users of this application.

**Comment by thomas-henn** [comment]
Behavior of Digital Twin Registry with respect to read access to digital twins is described here as aligned with the users of this application.

**Pull request merged**